code/modules/admin/view_variables/topic.dm

```diff
@@ -487,9 +487,9 @@
 	else if(href_list["remorgan"])
 		if(!check_rights(R_SPAWN))	return
 
-		var/mob/living/carbon/M = locate(href_list["remorgan"])
+		var/mob/living/carbon/human/M = locate(href_list["remorgan"])
```

Comment:
What is the purpose of making organ removal a mob proc if you're then going to restrict it to humans specifically?

Reply:
Surgery is coded that way. I can probably turn the check back to being a carbon. But I don't think organs work too well on carbon mobs currently. Plus I remember you mentioning the carbon subtype was possibly going away?

Reply:
done!